wolfcrypt/src/rsa.c

```diff
@@ -3546,7 +3547,8 @@ static int RsaPrivateDecryptEx(const byte* in, word32 inLen, byte* out,
             }
             return ret;
         }
-    #elif defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY)
+    #elif defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY) ||\
```

Comment:
For consistency, let's add a space after `||` before the backslash, for example `|| \`

Reply:
Thanks, Fixed.